react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -221,12 +221,12 @@ class AddPeopleDialog extends Component<*, *> {
             });
 
             this.props._conference
-                && this.props.inviteRooms(
+                && this.props.inviteVideoRooms(
                         this.props._conference,
                         this.state.inviteItems.filter(
                             i => i.type === 'videosipgw'));
 
-            invitePeople(
+            invitePeopleAndChatRooms(
```

Comment:
I just noticed, the other actions are put onto this.props but invitePeopleAndChatRooms isn't. Do you care?

Reply:
My mistake here. Damyan showed this is a function and shouldn't be mapped to props.